src/main/native/org/apache/commons/crypto/org_apache_commons_crypto.h

```diff
@@ -134,7 +134,7 @@ void *do_version_dlsym(JNIEnv *env, void *handle) {
 
 /* Microsoft C Compiler does not support the C99 inline keyword */
 #ifndef __cplusplus
-#define inline __inline;
+//#define inline __inline;
```

Comment:
What is the problem here? If this is broken then perhaps the whole thing should be deleted, instead of this line commented out.

Reply:
I had to comment out that line to get it to compile with MinGW-64.  Redefining the keyword causes a cascade of errors.  I left it commented as opposed to removing it completely because presumably some developer had to redefine inline to get it to compile under some other Microsoft compiler, and perhaps it would be helpful to keep it there for reference.